pyomeca/rototrans.py

```diff
@@ -1,499 +1,273 @@
-        """
-        return self.transpose()
+            size = (3, 1, 100)
+            angles = Angles.from_random_data(size=size)
+            angles_sequence = "xyz"
 
-    def mean(self):
+            rt = Rototrans.from_euler_angles(angles=angles, angle_sequence=angles_sequence)
+            ```
+
+            A translation vector can also be specified:
+
+            ```python
+            translation = Angles.from_random_data(size=size)
+            rt = Rototrans.from_euler_angles(
+                angles=angles, angle_sequence=angles_sequence, translations=translation
+            )
+            ```
         """
+        return rototrans.rototrans_from_euler_angles(
+            cls, angles, angle_sequence, translations
+        )
 
-        Returns
-        -------
-        Performs an optimization to compute the mean over the frames
+    @classmethod
+    def from_markers(
+        cls,
+        origin: xr.DataArray,
+        axis_1: xr.DataArray,
+        axis_2: xr.DataArray,
+        axes_name: str,
+        axis_to_recalculate: str,
+    ) -> xr.DataArray:
+        """
+        Rototrans DataArray from a specified set of markers.
+
+        Arguments:
+            origin: A marker constructed with `pyomeca.Markers()` corresponding
+                to the origin in the global reference frame
+            axis_1: Two markers that describe the first axis.
+                The first markers being the beginning of the vector and the second being the end.
+            axis_2: Two markers that describe the second axis.
+                The first markers being the beginning of the vector and the second being the end.
+            axes_name: Any combination of `x`, `y` and `z` describing the first and second axes.
+            axis_to_recalculate: Which of the two axes to recalculate
+
+        Returns:
+            Rototrans `xarray.DataArray` from the specified angles and angles sequence
+
+        !!! example
+            To create a system of axes from random markers:
+
+            ```python
+            from pyomeca import Markers, Rototrans
+
+            markers = Markers.from_random_data()
+
+            rt = Rototrans.from_markers(
+                origin=markers.isel(channel=[0]),  # first marker
+                axis_1=markers.isel(channel=[0, 1]),  # vector from the first and second markers
+                axis_2=markers.isel(channel=[0, 2]),  # vector from the first and third markers
+                axes_name="xy",  # axis_1 is x and axis_2 is y
+                axis_to_recalculate="y",  # we want to recalculate y
+            )
+            ```
         """
+        return rototrans.rototrans_from_markers(
+            cls, origin, axis_1, axis_2, axes_name, axis_to_recalculate
+        )
 
-        # Chose an arbitrary angle sequence to convert into angle during the optimization
-        seq = "xyz"
+    @classmethod
+    def from_transposed_rototrans(cls, rt: xr.DataArray) -> xr.DataArray:
```

Comment:
Again that is an intense name for effectively doing what on would expect by typing `.T` using a numpy array